### PR TITLE
add separator for raw format config

### PR DESCRIPTION
**Pull request by freeznet**
https://github.com/streamnative/pulsar-io-cloud-storage/issues/147

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,6 +51,7 @@ The Cloud Storage sink connector supports the following properties.
 | `sliceTopicPartitionPath` | Boolean| False |false | When it is set to `true`, split the partitioned topic name into separate folders in the bucket path. |
 | `withMetadata` | Boolean | False |false | Save message attributes to metadata. |
 | `withTopicPartitionNumber` | Boolean | False |true | When it is set to `true`, include topic partition number to the object path. |
+| `bytesFormatTypeSeparator` | String | False |"0x10" | Inserted between records for bytes formatType. Defaults to '0x10'. An input record that contains the line separator will look like multiple records in the output object. |
```
Comment (Huanli-Meng) [inline README.md:54]: Could we change it to :
It is inserted between records for the `formatType` of bytes. By default, it is set to '0x10'. An input record that contains the line separator looks like multiple records in the output object.

**Review by Huanli-Meng — COMMENTED** [review]
Leave a minor comment. PTAL

File: README.md (hunk 1 above)
Comment (freeznet) [reply README.md]: thanks, addressed in the latest commit

**Review by nlu90 — COMMENTED** [review]
Instead of having a customized separator (which to me is kind of error prone), why not wrap all the records inside an object and serialize this object instead?

**Comment by freeznet** [comment]
@nlu90 the separator is to solve the byte collision issue with less interference in the output. If we use a `List` or `Object` to wrap all the records, it will need the user to deserialize the structure first, then retrieve the records, which is not like a `raw` or `bytes` format.

**Comment by nlu90** [comment]
@freeznet Just curious to learn, whether it's possible that the selected separator will appear in the `raw` or `bytes` content, in which case, it's a collision also?

**Comment by freeznet** [comment]
@nlu90 yes, this is just the collision issue I talked about above. Before this PR, the separator is been set as `\n` (`0x10`), and it is super easy to get the collision issue  So we encourage the user to set a long separator to avoid the collision.

**Comment by nlu90** [comment]
To totally avoid the collision problem, is it possible to first write the length of the record bytes and then write the actual bytes? It has a scheme `lenght_byte:padding_bytes | lenght_byte:padding_bytes | lenght_byte:padding_bytes|`

**Comment by freeznet** [comment]
@nlu90 yes, nice solution, that will help and solve the collision issue, but should we introduce it as a new format type? since it changed the bytes format a lot.

**Review by nlu90 — APPROVED** [review]

**Pull request merged**